java/com/fortech/internship/service/impl/AddressService.java

```diff
@@ -0,0 +1,71 @@
+package com.fortech.internship.service.impl;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import com.fortech.internship.dao.AddressDAO;
+import com.fortech.internship.dto.AddressDTO;
+import com.fortech.internship.model.Address;
+
+
+@Service
+
+public class AddressService {
+	
+	@Autowired
+	private AddressDAO addressDAO;
+	@Autowired
+	private SessionFactory sessionFactory;
+
```

Comment:
We don't need to use sessionFactory in service

Reply:
solved, please check